### ISSUE TITLE
httpd

### ISSUE THREAD

**Issue by zhangsd20** [gl-inet/uboot-ipq40xx#1]
Hi:
   I see httpd related code in your code, I want to know if your uboot supports httpd to upgrade firmware ?
Thank you very much !

**Comment by lancersky** [comment]
yes it's.

**Comment by zhangsd20** [comment]
if i want to use your httpd code, what should i do?only use httpd/ ?

**Comment by lancersky** [comment]
Do you mean that you want to porting the code? Not just the httpd directory but also httpd.c and httpd.h in the net directory. The NetLoopHttpd entry is in net.c, you can searrch it. It's not difficult. you can do it.

**Comment by zhangsd20** [comment]
I ported your httpd and there is no problem with the functionality. But the led light is not controlled. I tried to read the register status before and after writing the register. It found that the status read after writing to GPIO_IN has not changed. The status bit0 read from GPIO_OUT has not changed. bit1 is 1. Because there is no datasheet, I am not clear about the ipq4019 register.

PS.I have only done mips porting before, its register operation is to write the function first and then write status (GPIO_OE_ADDRESS/GPIO_OUT_ADDRESS)
